trade-finance-manager-api/src/v1/rest-mappings/mappings/gef-facilities/mapGefFacilityDates.js

```diff
@@ -25,6 +25,16 @@ const mapGefFacilityDates = (facility, facilityTfm, dealSnapshot) => {
     mapped.coverEndDate = mapCoverEndDate(format(date, 'dd'), format(date, 'MM'), format(date, 'yyyy'), facility);
   }
 
+  mapped.facilityEndDateExists = facilityEndDateExists;
+
+  if (facilityEndDateExists) {
```

Comment:
I think @MarRobSoftwire has renamed this at my request :) might be worth catching up with him (should be something a bit easier to understand now for my little brain)